tests/attacks/lucash-pot-end.mcd

```diff
@@ -0,0 +1,93 @@
+// MCD Internal Contract Setup
+// ---------------------------
+
+// Auhthorize Pot/End for Vat
+transact ADMIN Vat . rely Pot
+transact ADMIN Vat . rely End
+
+// Authorize End for Cat/Vow/Pot
+transact ADMIN Cat . rely End
+transact ADMIN Vow . rely End
+transact ADMIN Pot . rely End
+
+// Authorize Vow for Flap/Flop
+transact ADMIN Flap . rely Vow
+transact ADMIN Flop . rely Vow
+
+// Initialize Vat accounts for Vow/Pot/Flap/End
+transact ADMIN Vat . initUser Vow
+transact ADMIN Vat . initUser Pot
+transact ADMIN Vat . initUser Flap
+transact ADMIN Vat . initUser End
+
+// File Vow contract for Pot (since Pot doesn't depend on Vow?)
```

Comment:
This is a `file` because the `Pot` needs to know where the `Vow` is so it can pass it through to the `Vat` when performing `drip`.

Reply:
ah makes sense